src/main/scala/centaur/api/CromwellClient.scala

```diff
@@ -33,6 +35,19 @@ object CromwellClient {
   final implicit val materializer: ActorMaterializer = ActorMaterializer(ActorMaterializerSettings(system))
   final val apiPath = "/api/workflows/v1"
 
+  def decodeResponse(response: HttpResponse): HttpResponse = {
+    val decoder = response.encoding match {
+      case HttpEncodings.gzip ⇒
+        Gzip
+      case HttpEncodings.deflate ⇒
+        Deflate
```

Comment:
Yep they have a ticket for automatic decoding https://github.com/akka/akka-http/issues/209